includes/admin/class-wc-admin-menus.php

```diff
@@ -53,14 +53,19 @@ public function __construct() {
 	public function admin_menu() {
 		global $menu;
 
-		$icon_svg = file_get_contents( WC()->plugin_url() . '/assets/images/classic-commerce-dashicon-white-on-transparent.svg' );
-		$cc_icon = base64_encode( $icon_svg );
+		$cc_icon = '';
+		$icon_file = WC_ABSPATH . 'assets/images/classic-commerce-dashicon-white-on-transparent.svg';
+
+		if ( file_exists( $icon_file ) ) {
```

Comment:
Under what circumstances would this file not exist? I'm not sure if this is strictly necessary but it's not going to do any real harm either.